packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -490,6 +490,16 @@ export class RemoteMediaManager extends EventsScope {
     return this.currentLayoutId;
   }
 
+  /**
+   * sets the preferLiveVideo
+   */
+  public setPreferLiveVideo(preferLiveVideo: boolean) {
+    this.config.video.preferLiveVideo = preferLiveVideo;
+    Object.values(this.media.video.activeSpeakerGroups).forEach((activeSpeakerGroup) => {
+      activeSpeakerGroup.setPreferLiveVideo(preferLiveVideo);
```

Comment:
once you add the commit parameter to `RemoteMediaGroup.setPreferLiveVideo()` we will need to call it with commit being `false` here and after the loop call `this.mediaRequestManagers.video.commit();`